docs/specs/xref.yml

```diff
@@ -147,11 +147,13 @@ inputs:
     }
   docs/c.md: |
     Link to <xref:a?displayProperty=fullName>
+    Link to <xref:a??displayProperty=fullName>
     Link to @a?displayProperty=fullName
+    Link to @a??displayProperty=fullName
     Link to <xref:a?displayProperty=notExisting>
 outputs:
   docs/c.json: | 
-    {"conceptual":"<p>Link to <a href=\"a.json\">test.test</a>\nLink to <a href=\"a.json\">test.test</a>\nLink to <a href=\"a.json\">test</a></p>\n"}
+    {"conceptual":"<p>Link to <a href=\"a.json\">test.test</a>\nLink to <a href=\"a.json\">test</a>\nLink to <a href=\"a.json\">test.test</a>\nLink to <a href=\"a.json\">test</a>??displayProperty=fullName\nLink to <a href=\"a.json\">test</a></p>\n"}
```

Comment:
why would we want behaviors are different between `xref` and `@`

Reply:
Because @uid??displayProperty, the href in xref block is `uid`
and <xref: uid??displayProperty>, the href in xref block is `uid??displayProperty`

Reply:
https://github.com/dotnet/docfx/blob/cc1757a8e01658b2de49a8dfa135ec7ccd7c7ea4/src/docfx/lib/markdown/ResolveXrefExtension.cs#L29
I am talking about this one

Reply:
This shares the same behavior with v2.

Reply:
I see, this is because shorthand syntax match does not allow consecutive symbols in `XrefInlineShortParser`